### PR TITLE
Implement Toggle Task Function

### PR DESCRIPTION
**Pull request by simonteozw**

### CODE REVIEW

File: src/main/java/trackitnus/logic/commands/task/ToggleTaskCommand.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import trackitnus.logic.commands.exceptions.CommandException;
+import trackitnus.model.Model;
+import trackitnus.model.task.Task;
+
+public class ToggleTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "toggle";
+
+    public static final String MESSAGE_USAGE = Task.TYPE + " " + COMMAND_WORD
+        + ": Toggles completion status of the task identified by the index number used in the displayed task list.\n"
```
Comment (UncleGrandpa925) [inline src/main/java/trackitnus/logic/commands/task/ToggleTaskCommand.java]: Can you look at Minh's new messages and standardise the message content?

File: src/main/java/trackitnus/model/task/Task.java [hunk 2]
```diff
@@ -119,8 +124,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
+        String completionStatus = isDone ? "[DONE]" : "[NOT DONE]";
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
+            .append(completionStatus)
             .append(" Date: ")
             .append(getDate());
```
Comment (UncleGrandpa925) [inline src/main/java/trackitnus/model/task/Task.java]: It should be in the form of [DONE] + Name + Date: .... right?

**Review by UncleGrandpa925 — COMMENTED** [review]
Nicely done! I just have some minor comments 😄

File: src/main/java/trackitnus/model/task/Task.java (hunk 2 above)
Comment (simonteozw) [reply src/main/java/trackitnus/model/task/Task.java]: Hmm yes I suppose. Will change it

**Review by UncleGrandpa925 — APPROVED** [review]

**Pull request merged**